app/services/autoscale-formula.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from "@angular/core";
+import { AutoscaleFormula } from "app/models";
+import { Constants, log } from "app/utils";
+import * as storage from "electron-json-storage";
```

Comment:
I think we also have to update ssh-key service as well. That's code I originally cloned from.